fmlearlydisplay/src/test/java/net/minecraftforge/fml/earlydisplay/tests/DisplayTest.java

```diff
@@ -0,0 +1,99 @@
+package net.minecraftforge.fml.earlydisplay.tests;
+
+import net.minecraftforge.fml.earlydisplay.*;
+import net.minecraftforge.fml.loading.progress.StartupNotificationManager;
+import org.junit.jupiter.api.Test;
+import org.lwjgl.opengl.GL;
+
+import java.util.List;
+
+import static org.lwjgl.glfw.GLFW.*;
+import static org.lwjgl.opengl.GL11.*;
+
+public class DisplayTest {
+    @Test
+    void testFont() throws InterruptedException {
+//        System.setProperty("java.awt.headless", "true");
```

Comment:
bunch of commented code

Reply:
Yeah it was my test harness for trying stuff out.